.gitignore

```diff
@@ -1,2 +1,11 @@
 /target/
 /src/main/resources/local.properties
+vagrant/ansible/files/qucosa-fcrepo-reportingdb-1.0.0-SNAPSHOT.war
+.classpath
+.project
+.settings/
```

Comment:
.classpath, .project, build.properties and .settings/ are IDE specific folder and should not be excluded here.
Better to configure these in your global .gitignore configuration:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore